docs/utils/expr.js

```diff
@@ -0,0 +1,23 @@
+// Quick one-off utility to convert a Column Expression into a single-line
```

Comment:
The easiest way to do this is to click the `Copy` button in the `<perspective-viewer>` status bar, and select `.config.json` option to copy the current `<perspective-viewer>` config to the clipboard as JSON.

Regardless, this file shouldn't be in the PR.